### PR TITLE
Tests/tests jvbb

### PR DESCRIPTION
**Pull request by JVBB83**
Made first test for a Brain Damaged Operator

### CODE REVIEW

File: BusinessCore.Tests/JVBBTests.cs [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+
+namespace BusinessCore.Tests
+{
+    [TestClass]
+    [TestCategory("JVBB Tests")]
+    public class JVBBTests
+    {
+        [TestMethod]
+        public void BrainDamagedBuilder_01()
```
Comment (hossmi) [inline BusinessCore.Tests/JVBBTests.cs]: test method name must be more descriptive, please :)

File: BusinessCore.Tests/JVBBTests.cs [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+            Negassert.mustFail(() => vehicle.Engine.start());
+
+            Negassert.mustFail(() => vehicle.setWheelsPressure(-1));
+            vehicle.setWheelsPressure(0);
+            vehicle.setWheelsPressure(0.99);
+            vehicle.setWheelsPressure(2);
+
+        }
+
+        //[TestMethod]
```
Comment (hossmi) [inline BusinessCore.Tests/JVBBTests.cs]: Comments? no comments please :)

File: BusinessCore.Tests/JVBBTests.cs (hunk 1 above)
Comment (JVBB83) [reply BusinessCore.Tests/JVBBTests.cs]: How about WrongBuildRequests for the method name?

File: BusinessCore.Tests/JVBBTests.cs (hunk 2 above)
Comment (JVBB83) [reply BusinessCore.Tests/JVBBTests.cs]: I shall remove them.

**Pull request merged**